app/assets/javascripts/posts/editor.js

```diff
@@ -253,29 +253,29 @@ function bindIcon() {
   });
 }
 
-function galleryString(gallery, multiGallery) {
-  var iconsString = "";
+function galleryNode(gallery, multiGallery) {
+  var iconNodes = [];
   var icons = gallery.icons;
 
   for (var i=0; i<icons.length; i++) {
-    iconsString += iconString(icons[i]);
+    iconNodes.push(iconNode(icons[i]));
   }
 
-  if (!multiGallery) return iconsString;
+  if (!multiGallery) return iconNodes;
 
-  var nameString = "<div class='gallery-name'>" + gallery.name + "</div>";
-  return "<div class='gallery-group'>" + nameString + iconsString + "</div>";
+  var nameNode = $("<div class='gallery-name'></div>").text(gallery.name);
+  return $("<div class='gallery-group'>").append(nameNode).append(iconNodes);
```

Comment:
do we need a </div> here?

Reply:
Shouldn't need to! I'll add it anyway but I think jQuery does that automatically

Reply:
```suggestion
  return $("<div class='gallery-group'></div>").append(nameNode).append(iconNodes);
```

Reply:
Just wasn't sure because the line above had one; don't really care which way we make it consistent but they should probably match